node/src/Consumer.ts

```diff
@@ -154,9 +154,27 @@ export type ConsumerStat =
  */
 export type ConsumerType = 'simple' | 'simulcast' | 'svc' | 'pipe';
 
+type ObserverEvents = {
+	close: [];
+	pause: [];
+	resume: [];
+	score: [ConsumerScore];
+	layerschange: [ConsumerLayers?];
+	trace: [ConsumerTraceEventData];
+}
+
+type ConsumerEvents = Pick<ObserverEvents, 'score' | 'layerschange' | 'trace'>
```

Comment:
Even if Consumer and Consumer.observer events sometimes match, can we repeat their definition instead of picking some of them from the other?

Reply:
fixed